js/count_monkeys.test.js

```diff
@@ -0,0 +1,26 @@
+const {
+  monkeyCountDoWhile,
+  monkeyCountForLoop,
+  monkeyCountFrom,
+} = require('./count_monkeys');
+
+const functionsToTest = [
+  monkeyCountForLoop,
+  monkeyCountFrom,
+  monkeyCountDoWhile,
+];
+
+functionsToTest.forEach((functionsToTest) => {
+  describe(`count_monkeys ${functionsToTest} function`, () => {
```

Comment:
Did you run the tests? What are the test names? They do not look so friendly. Somehow extracting only the name of the function, instead of the whole implementation

Reply:
![Zrzut ekranu 2020-01-21 o 23 27 15](https://user-images.githubusercontent.com/15020706/72848790-a4c70c00-3ca5-11ea-9746-e134ee81af3c.png)
now I think its done right way